server/cache.go

```diff
@@ -548,12 +548,12 @@ func (s *storeInfo) usedRatioScore() int {
 }
 
 // leaderScore is the leader peer count score of store, the score range is [0,100].
-func (s *storeInfo) leaderScore(regionCount int) int {
+func (s *storeInfo) leaderScore(leaderCount int, regionCount int) int {
```

Comment:
This function has nothing to do with storeInfo now. You can move it to balancer.go.